### PR TITLE
Debug renderers mappings

### PR DESCRIPTION
**Pull request by IotaBread**
Names almost everything remaining related to debug renderers. `StructureDebugRenderer.f_cyiqepya` still needs a name, it is a map that holds whether a structure piece is a start piece

### CODE REVIEW

**Comment by OroArmor** [comment]
oh i did a bunch of these in #5... oops

**Comment by IotaBread** [comment]
Oh...

File: mappings/net/minecraft/client/render/debug/NeighborUpdateDebugRenderer.mapping [hunk 1]
```diff
@@ -6,3 +6,5 @@ CLASS net/minecraft/unmapped/C_kinexwyn net/minecraft/client/render/debug/Neighb
 	METHOD m_fqsnafbz addNeighborUpdate (JLnet/minecraft/unmapped/C_kkdaznyh;)V
 		ARG 1 time
 		ARG 3 pos
+	METHOD m_uaajzpqx (Ljava/lang/Long;)Ljava/util/Map;
+		ARG 0 _long
```
Comment (LambdAurora) [inline mappings/net/minecraft/client/render/debug/NeighborUpdateDebugRenderer.mapping]: This feels strange

**Review by LambdAurora — APPROVED** [review]

**Review by NoComment1105 — APPROVED** [review]

**Review by OroArmor — APPROVED** [review]

**Comment by OroArmor** [comment]
should this be targeting 21w39a?

**Comment by IotaBread** [comment]
Idk if something has changed in 21w39a since the last snapshot, I made the mappings in 1.17.1

**Pull request merged**